design/0001-metallb-crd.md

```diff
@@ -0,0 +1,251 @@
+# MetalLB Custom Resource Controller
```

Comment:
Somewhere in here can you add a link to #196 ?  That's the issue opened a few years ago to track this idea.

Reply:
it added in the description section of this PR